app/src/main/java/com/monkeyteam/chimpagne/ui/navigation/NavigationActions.kt

```diff
@@ -16,6 +16,7 @@ object Route {
   const val MY_EVENTS_SCREEN = "myEvents"
   const val VIEW_DETAIL_EVENT_SCREEN = "viewDetailEventScreen"
   const val MANAGE_STAFF_SCREEN = "manageStaffScreen"
+  const val SUPPLIES_SCREEN = "SuppliesScreen"

```

Comment:
the previous values use a lowercase character as the first character

Reply:
It seems a bit anarchic 
![image](https://github.com/Chimpagne/ChimpagneApp/assets/39198541/3bfd3f15-47dc-4ca1-a809-563b78d9ccde)
A future PR will solve this issue.